scripts/e2e-test-regions.json

```diff
@@ -0,0 +1,21 @@
+[
+  "ap-northeast-2",
```

Comment:
nit: Please alphabetize this list so it's easier to scan for missing/new regions in future.

Slightly bigger nit: If we're already maintaining a list of supported regions somewhere else in this codebase, let's reuse that rather than maintaining another source of truth for the supported regions.

Reply:
Changed the format to include the region optionality info and sorted by region name. We previously had 2 lists in cleanup, split-e2e-tests scripts but with this change I have consolidated them into a single file.